### ISSUE TITLE
[instagram-feed] Instagram Basic Display API Deprecation 

### ISSUE THREAD

**Issue by andreas-albers** [getgrav/grav-premium-issues#461]
Hey Grav Dev Team,

As far as my understanding goes, the Instagram feed plugin utilizes the [Instagram basic display API](https://developers.facebook.com/docs/instagram-basic-display-api/). This API will be deprecated on December 4th.
Are any updates planned to keep this plugin running after December 4th (e.g. via the Instagram API)?

Best
Andreas

**Comment by rhukster** [comment]
Yes as mentioned here: https://github.com/getgrav/grav-premium-issues/issues/460#issuecomment-2395467299 i'm aware of this issue, but still have yet to find to do what we need to with the regular API.  The espresso one (https://github.com/espresso-dev/instagram-php) suffers from the same issue, it relies on Basic Display.

**Comment by andreas-albers** [comment]
Thanks @rhukster for the information.
Maybe I'm missing something, but there are to espresso repos:
[This old](https://github.com/espresso-dev/instagram-basic-display-php) version for the Basic Display API. Due to the deprecation of the Basic Display API, the readme of this repo refers to [this new](https://github.com/espresso-dev/instagram-php) version for the Instagram API. However, the readme of the new repo mentions the Basic Display API, which might be a typo, but is definitely confusing.
So maybe the new repo might work but I cannot judge that myself.
Nevertheless, thanks for your effort :)

**Comment by andreas-albers** [comment]
Since my question has been answered, this issue can be closed, as far as I'm concerned.

**Comment by rhukster** [comment]
Closed due to inactivity
